src/components/ButtonDropdown/ButtonDropdown.stories.tsx

```diff
@@ -59,3 +61,36 @@ export const PositionBottomRight: StoryObj<Args> = {
     position: 'bottom-right',
   },
 };
+
+const TestComponent = ({ color }: { color: string }) => {
+  return (
+    <>
+      <DropdownMenuItem>
+        <Icon name="schedule" purpose="decorative" size="1.25rem" />
+        Item 1
+      </DropdownMenuItem>
+      <DropdownMenuItem>
+        <div style={{ backgroundColor: color }}>
+          <Icon name="schedule" purpose="decorative" size="1.25rem" />
+          Item 2
+        </div>
+      </DropdownMenuItem>
+      <DropdownMenuItem>
+        <Icon name="schedule" purpose="decorative" size="1.25rem" />
+        Item 3
+      </DropdownMenuItem>
+      <DropdownMenuItem>
+        <div style={{ backgroundColor: color }}>
+          <Icon name="schedule" purpose="decorative" size="1.25rem" />
+          Item 4
+        </div>
+      </DropdownMenuItem>
+    </>
+  );
+};
+
+export const ComponentWrapped: StoryObj<Args> = {
+  args: {
+    children: <TestComponent color="blanchedalmond" />,
+  },
+};
```

Comment:
This story emulates having wrapping helper components where arrow key navigation was previously not working. Confirmed not working by cherry-picking onto `next`, and confirmed working here.